### PR TITLE
Add `new featured artist: bill wurtz` news post

### PR DESCRIPTION
**Pull request by pishifat**
links

### CODE REVIEW

File: news/2022-02-02-new-featured-artist-bill-wurtz.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+[piano beatmap](https://osu.ppy.sh/beatmapsets/735580) by [Nivrad00](https://osu.ppy.sh/users/1984634)
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### and the day goes on
+
+[nothing beatmap](LINK)
```
Comment (Nivalyx) [inline news/2022-02-02-new-featured-artist-bill-wurtz.md:35]: would greatly prefer `beatmap template 1` and `beatmap template 2` over `nothing beatmap 1` and `nothing beatmap 2` tbh, "nothing beatmap" just... doesn't feel quite right :(

File: news/2022-02-02-new-featured-artist-bill-wurtz.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+[nothing beatmap 2](LINK)
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+---
+
+beatmap templates: [**bill wurtz**'s featured artist listing](https://osu.ppy.sh/beatmaps/artists/236)
```
Comment (Nivalyx) [inline news/2022-02-02-new-featured-artist-bill-wurtz.md:51]: no mention of @osugame twitter?

File: news/2022-02-02-new-featured-artist-bill-wurtz.md (hunk 1 above)
Comment (pishifat) [reply news/2022-02-02-new-featured-artist-bill-wurtz.md:35]: i think it fits the style better as is (going from circles to piano to nothing)

File: news/2022-02-02-new-featured-artist-bill-wurtz.md (hunk 2 above)
Comment (pishifat) [reply news/2022-02-02-new-featured-artist-bill-wurtz.md:51]: *minimalist*

**Review by Walavouchey — DISMISSED** [review]

**Review by peppy — APPROVED** [review]

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**